client/src/Components/Payment/CardInfoElement.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { useStripe, useElements, CardElement } from "@stripe/react-stripe-js";
+import { Button } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { Alert } from "@material-ui/lab";
+import axios from "axios";
+import { Redirect } from "react-router-dom";
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    border: "1px solid #d3d3d3",
+    padding: "10px",
+    margin: "0 0 10px 0",
+    borderRadius: "5px",
+  },
+}));
+
+const CardInfoElement = () => {
+  const classes = useStyles();
+  const stripe = useStripe();
+  const elements = useElements();
+  const [error, setError] = useState("");
+  const [load, setLoad] = useState(false);
+  const [success, setSuccess] = useState(false);
+
+  const handleSubmit = async (event) => {
+    // We don't want to let default form submission happen here,
+    // which would refresh the page.
+    setLoad(true);
+    setError("");
+    event.preventDefault();
+
+    if (!stripe || !elements) {
```

Comment:
not super important but couldn't you just set `disabled={!stripe || !elements}` down in the `<Button>` component and get rid of these lines?